src/components/select/select.js

```diff
@@ -1122,12 +1140,12 @@ function SelectProvider($$interimElementProvider) {
       function watchAsyncLoad() {
         if (opts.loadingAsync && !opts.isRemoved) {
           scope.$$loadingAsyncDone = false;
-          scope.progressMode = 'indeterminate';
+          scope.$mdProgressMode = 'indeterminate';
```

Comment:
It's to make sure that we don't overwrite anything within the user scope (because we are now using the same one). `$md-` variables should not be used in anyone apps.